src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/unusedlocalvariable/InputUnusedLocalVariableNestedClasses3.java

```diff
@@ -0,0 +1,59 @@
+/*
+UnusedLocalVariable
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.unusedlocalvariable;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Target;
+
+public class InputUnusedLocalVariableNestedClasses3 {
+    void method() {
+        final OuterClass outerClass = new OuterClass(); // ok
+        final OuterClass.InnerClass innerClass = outerClass.new InnerClass();
+        System.out.println(innerClass);
+    }
+
+    class OuterClass {
+        class InnerClass {
+            InnerClass() { }
+        }
+    }
+
+
+}
+
+class Outer {
+    public class Inner { }
+}
+
+class Test {
+    void m() {
+        @A Outer p1 = new @A Outer(); // ok
+        @A Outer.@B Inner p2 = p1.new @B Inner(); // violation
```

Comment:
please append message after `// violation` 
https://github.com/checkstyle/checkstyle/blob/6fee303a364eaf903039c3227801bda8a09c3dba/src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/methodcount/InputMethodCountInterfaceMemberScopeIsPublic.java#L27

sorry that we did not finish migration of all tests and there are bunch of bad examples.

Reply:
https://github.com/checkstyle/checkstyle/blob/6fee303a364eaf903039c3227801bda8a09c3dba/src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/rightcurly/InputRightCurlyTestOptionAlone.java#L80-L81

Reply:
@nrmancuso , @pbludov , please keep reminding about this contributors.

Reply:
Done